server/src/utils/cache.ts

```diff
@@ -0,0 +1,3 @@
+const NodeCache = require( "node-cache" );
```

Comment:
import

Reply:
import * as NodeCache from 'node-cache';

Reply:
seems to be working this way